src/main/kotlin/no/nav/syfo/aktivitetskrav/api/ForhandsvarselDTO.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.syfo.aktivitetskrav.api
+
+import no.nav.syfo.aktivitetskrav.domain.AktivitetskravStatus
+import no.nav.syfo.aktivitetskrav.domain.AktivitetskravVurdering
+import java.time.LocalDate
+
+data class ForhandsvarselDTO(
+    val fritekst: String,
+    val document: List<DocumentComponentDTO> = emptyList(),
+) {
+    fun toAktivitetskravVurdering(veilederIdent: String) = AktivitetskravVurdering.create(
+        status = AktivitetskravStatus.FORHANDSVARSEL,
+        createdBy = veilederIdent,
+        beskrivelse = this.fritekst,
+        arsaker = emptyList(),
+        // Svarfrist for innbygger å svare på forhandsvarsel.
+        // Oppgave lages for veileder når denne fristen har gått ut
+        // Sjekk om dette er riktig. Forskjellig frist når fysisk brev og digitalt?
+        frist = LocalDate.now().plusWeeks(2),
```

Comment:
Her må vi finne utav hvilken frist som gjelder. Både for fysisk brevutsendelse og digitalt, med mindre disse 2 fristene er de samme.

Reply:
Hvordan vil vi håndtere at en cronjob har plukket opp denne fristen? Hvis vi vil at cronjobben skal kjøre flere ganger samme dag må vi legge inn en mekanisme for at denne fristen ikke blir plukket opp flere ganger samme dag.